.swiftlint.yml

```diff
@@ -0,0 +1,40 @@
+disabled_rules:
+- nesting
+- unused_closure_parameter
```

Comment:
Why was this one added?

Reply:
nesting per our slack conversation (//Nesting Violation: Types should be nested at most 1 level deep (nesting))

unused_closure_parameter was a mistake. It needed to be aded while the auto corrector was running. it has been removed.